core/src/main/res/raw/ayp_youtube_player.html

```diff
@@ -162,5 +162,9 @@
             player.setPlaybackRate(playbackRate);
         }
 
+        function toggleFullscreen() {
+            player.toggleFullscreen();
```

Comment:
Is this function documented somewhere in the IFrame Player API? how did you find out about it?

Reply:
I have done some research, that method shows up when `origin` is set to `www.youtube.com`. Therefore I am unsure about how reliable that API is.

Can we try to use [this dom API](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen) to enter full screen instead?

Reply:
Refactored code as per suggestion, use dom api for enter and exit fullscreen mode. Please help to review.